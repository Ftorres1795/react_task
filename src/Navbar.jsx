function Navbar() {
  return (
    <header className='navbar bg-success text-white'>
      <nav className='d-flex'>
        <span className='header-text fs-3'>Task List</span>
        <sub className='sub-header-text'>Rest API</sub>
      </nav>
    </header>
  );
}
export default Navbar;
