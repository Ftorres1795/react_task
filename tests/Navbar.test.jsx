import { describe, expect, it } from "vitest";
import { render } from "@testing-library/react";

import Navbar from "../src/Navbar";

describe("Test for Navbar component", () => {
  it("should match snapshot", () => {
    const { container } = render(<Navbar />);
    expect(container).toMatchSnapshot();
  });
});
