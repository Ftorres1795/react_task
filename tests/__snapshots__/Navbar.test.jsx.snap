// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test for Navbar component > should match snapshot 1`] = `
<div>
  <header
    class="navbar bg-success text-white"
  >
    <nav
      class="d-flex"
    >
      <span
        class="header-text fs-3"
      >
        Task List
      </span>
      <sub
        class="sub-header-text"
      >
        Rest API
      </sub>
    </nav>
  </header>
</div>
`;
